web/queries/organization.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Community.Queries.Organization do
+  alias Community.Organization
+  alias Community.Repo
+
+  def find_organization do
+    Organization
+    |> Repo.one
+    |> case do
+      nil ->
+        %Organization{}
```

Comment:
This is an interesting decision. Can you explain your thoughts? This seems like the kind of thing that should be an exception was my thought but I'd love to hear why you went this route.

Reply:
When someone deploys their version of Community, they will have an empty database, and the app will crash. This will set them up with a placeholder Organization since the app now assumes there will be an organization.

Reply:
I could instead put this into a mix task to prime the database for new deployments-- but this was also just as effective and didn't need another step.